src/table/sortable-head-cell.js

```diff
@@ -28,25 +28,33 @@ function SortDirectionIcon({direction}: {direction: SortDirectionT}) {
 }
 
 export default function SortableHeadCell(props: HeadCellPropsT) {
-  const {overrides = {}} = props;
+  const {overrides = {}, extendClick} = props;
 
   const [HeadCell, headCellProps] = getOverrides(
     overrides.HeadCell,
     StyledHeadCell,
   );
+
   const [SortableLabel, sortableLabelProps] = getOverrides(
     overrides.SortableLabel,
     StyledSortableLabel,
   );
 
+  const onClick = () => {
+    props.onSort && props.onSort();
+  };
+
   return (
-    <HeadCell role="columnheader" {...headCellProps}>
+    <HeadCell
+      role="columnheader"
```

Comment:
Can you please create an e2e tests that validates this has no accessibility issues? Here is an example: https://github.com/uber-web/baseui/blob/master/src/checkbox/__tests__/checkbox.e2e.js#L17-L21 I'm believe applying an `onClick` property to an element without `role="button"` will not properly translate the meaning to screen readers

Reply:
Added e2e tests which didn't show any accessibility issues. You are right, adding `onClick` does not translate that to screen readers but screenreader behaves similar to the case with no fillClickTarget which is accessible. Validated the behavior with voiceover on chrome. Please suggest.